aws/sdk/examples/telephone-game/src/main.rs

```diff
@@ -0,0 +1,439 @@
+            // Collect user input
+            let mut input_lines = BufReader::new(tokio::io::stdin()).lines();
+
+            // Reading a line from stdin will include the trailing newline
+            // We don't want that so we trim it
+            line = input_lines
+                .next_line()
+                .await
+                .expect("encountered IO error when attempting to read next line")
+                .expect("stream is closed")
+                .trim()
+                .to_owned();
+
+            // If the user types "quit" then the game will end
+            if line == "quit" {
+                break;
+            }
+
+            // Start synthesizing speech and wait for it to finish
+            let res = polly_client
+                .synthesize_speech()
+                .text(&line)
+                .voice_id(VoiceId::Joanna)
+                .output_format(OutputFormat::Mp3)
+                .send()
+                .await;
+
+            match res {
+                Ok(res) => {
+                    info!(r#"You said: "{}""#, &line);
+                    info!("Playing Polly's response...");
+
+                    // Collect the ByteStream returned by the synthesize_speech call
+                    let byte_stream = res
+                        .audio_stream
+                        .collect()
+                        .await
+                        .expect("audio stream ended prematurely");
+
+                    // Create a file to store the audio
+                    let mut tmp_file = tokio::fs::File::create(&tmp_file_path)
+                        .await
+                        .expect("couldn't create temp file");
+                    // Write the ByteStream to the file
+                    tmp_file
+                        .write_all(&byte_stream.into_bytes())
+                        .await
+                        .expect("couldn't write to temp file");
+                    // Flush the write operation to ensure it finishes before we continue
+                    tmp_file.flush().await.expect("couldn't flush");
+
+                    // Open the audio file with regular blocking IO File
+                    // rodio's Decoder requires stdlib Files
+                    let file = std::fs::File::open(&tmp_file_path).unwrap();
+                    let source =
+                        Decoder::new(std::io::BufReader::new(file)).expect("couldn't decode audio");
+
+                    // Set rodio to play the audio we just decoded
+                    sink.append(source);
+                    sink.sleep_until_end();
+
+                    info!("Did you hear it?");
+                }
+                Err(e) => error!("{}", e),
+            };
+
+            // Clear last iteration's line so we can receive new user input
+            line.clear();
+        }
+    })
+}
+
+
+/**
+# Play a game of Telephone w/ AWS
+
+1. Fetch user any user input that will override default values
```

Comment:
was this intended to be in a doc comment?

Reply:
yes, lol

My javascript is showing